tools/static-assets/server/boot.js

```diff
@@ -285,7 +285,19 @@ var loadServerBundles = Profile("Load server bundles", function () {
           }
         }
 
-        var resolved = require.resolve(name);
+        try {
+          var resolved = require.resolve(name);
+        } catch (err) {
+          const nameParts = name.split("/");
+          // If it looks like `meteor/xxx`, the user may forgot to add the
```

Comment:
I think we can drop this comment; this code pretty much speaks for itself (especially since the error message outlines the case we're handling here).